### DISCUSSION TITLE
[windows] Support installation from scoop.sh

### DISCUSSION THREAD

**Discussion by gianluca-sabena** [spacedriveapp/spacedrive#1711] [Feature Request]
<https://scoop.sh/> is a open source windows package manager, I suggest to add spacedrive to the list of sccop applications

**Comment by Bobrokus** [comment]
+1
This would be more than awesome

**Comment by HeavenVolkoff** [comment]
Hi, at the moment we don't have plans to support scoop. However we are open to PRs from the community adding support for it.

Also, Spacedrive can be installed through winget:
`$> winget install --id spacedrive.Spacedrive`
